src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -123,21 +123,20 @@ class SourcesTree extends Component<Props, State> {
     }
 
     if (nextProps.shownSource && nextProps.shownSource != shownSource) {
-      const listItems = getDirectories(nextProps.shownSource, sourceTree);
-
-      if (listItems && listItems[0]) {
-        this.selectItem(listItems[0]);
-      }
-
-      return this.setState({ listItems });
+      // const listItems = getDirectories(nextProps.shownSource, sourceTree);
```

Comment:
will figure out how to make this a source tomorrow...

not sue why selectItem fires selectSource again, that is a bit weird...